scripts/p2p-test-network.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+## Update P2P test network if correct repo and branch.
+set -eo pipefail
+
+SSH_USERNAME="root"
+
+# Tag and push rnode docker container when it meets criteria.
+if [[ "${TRAVIS_BRANCH}" = "master" || \
+      "${TRAVIS_BRANCH}" = "dev" || \
+      "${TRAVIS_BRANCH}" = "ops-test" ]] \
+    && [[ "${TRAVIS_REPO_SLUG}" = "rchain/rchain" ]] ; then
+    #&& [[ "${TRAVIS_PULL_REQUEST}" = "false" && "${TRAVIS_REPO_SLUG}" = "rchain/rchain" ]] ; then
+
+    echo "Travis branch ${TRAVIS_BRANCH} matched and from repo rchain/rchain. Pushing rnode to Docker repo."
+
+    # Generate rnode debian and rpm packages - push to repo and then to p2p test net
+    sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate node/rpm:packageBin node/debian:packageBin 
+    scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -P 10003 \
```

Comment:
`StrictHostKeyChecking=no` is basically `AllowManInTheMiddle=yes`. Why not explicitly add the host key in the script?